### PR TITLE
Value-based tests for toy gaussian problem

### PR DESCRIPTION
**Pull request by MichaelClerx**
As part of #720

### CODE REVIEW

**Comment by MichaelClerx** [comment]
@ben18785 It needs to pass the unit tests though :D :D :D

**Comment by MichaelClerx** [comment]
(Also, its your code, so somebody else should be reviewing. I just opened a PR to get this branch out in the open!)

**Comment by ben18785** [comment]
@MichaelClerx If you're happy with this, can we merge please?

File: pints/tests/test_toy_gaussian_logpdf.py [hunk 1]
```diff
@@ -132,6 +137,24 @@ def test_gaussian_sensitivity(self):
         self.assertAlmostEqual(L, -25.10903637045394)
         self.assertTrue(np.array_equal(dL, [0.25, 3.5, 3.0]))
 
+        # check incorrect covariance matrices being added
+        self.assertRaises(ValueError,
+                          pints.toy.GaussianLogPDF, [1, 2], [[1, 0.5],
+                                                             [-0.5, 2]])
+        self.assertRaises(ValueError,
```
Comment (MichaelClerx) [inline pints/tests/test_toy_gaussian_logpdf.py]: It's a personal preference, but I'd have written these as
```
self.assertRaises(
    ValueError, ...)
```
Just out of interested, do you find the current form prettier / easier to read?

**Pull request merged**